remindme-api/src/main/java/it/andreafailli/remindme/api/auth/FirebaseAuthenticationToken.java

```diff
@@ -0,0 +1,52 @@
+package it.andreafailli.remindme.api.auth;
+
+import java.util.Collection;
+
+import org.springframework.security.authentication.AbstractAuthenticationToken;
+import org.springframework.security.core.GrantedAuthority;
+
+import com.google.firebase.auth.FirebaseToken;
+
+public class FirebaseAuthenticationToken extends AbstractAuthenticationToken {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Override the "equals" method in this class. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2160)